cards/event-standard/template.hbs

```diff
@@ -52,7 +52,7 @@
         </div>
         {{/if}}
         {{#if card.feedback}}
-          {{> thumbsfeedback card cardType="HitchhikerEventStandard" feedbackSubmitted=feedbackSubmitted}}
+          {{> thumbsfeedback card cardType="HitchhikerCard" feedbackSubmitted=feedbackSubmitted}}
```

Comment:
maybe we shouldn't call this cardType anymore? Alternatively we could just always have the `HitchhikerCard-[blah]` css class on the feedback partial and only have special styling for the ones that currently have cardType="not HitchhikerCard"

Reply:
I'll change that